app/identifiers/LastPageId.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
```

Comment:
Not needed

Reply:
discussed with Paul and creating another ticket to refactor in all services